internal/lsp/completion.go

```diff
@@ -306,9 +306,9 @@ func newAllPortTypesFromModule(suite *ntt.Suite, modName string, sortPref string
 	portList := make([]protocol.CompletionItem, 0, len(items))
 	for _, v := range items {
 		if len(sortPref) > 0 {
-			portList = append(portList, protocol.CompletionItem{Label: v, Kind: protocol.StructCompletion, SortText: sortPref + v, Detail: modName + "." + v})
+			portList = append(portList, protocol.CompletionItem{Label: v, Kind: protocol.InterfaceCompletion, SortText: sortPref + v, Detail: modName + "." + v})
```

Comment:
This change seems unrelated. Has it been pushed on purpose or by accident?

Reply:
it is unrelated